python/templates/Collection.h.jinja2

```diff
@@ -203,8 +203,8 @@ private:
   bool m_isValid{false};
   mutable bool m_isPrepared{false};
   bool m_isSubsetColl{false};
-  uint32_t m_collectionID{0};
-  mutable std::unique_ptr<std::mutex> m_storageMtx{nullptr};
+  uint32_t m_collectionID{static_cast<uint32_t>(podio::ObjectID::untracked)};
```

Comment:
This changes the behavior. I think it's the right thing to do, but we should probably declare this at least in the release notes.

Reply:
In which case? Both constructors were initializing to `podio::ObjectID::untracked`, so it was impossible to get 0.